### DISCUSSION TITLE
XAMLIL Root element is missing

### DISCUSSION THREAD

**Discussion by githubiyo** [AvaloniaUI/Avalonia#8244] [Q&A]
Failed to start an avaloniaUI application:
Error info:
Severity	Code	Description	Project	File	Line	Suppression State
Error	XAMLIL	Root element is missing.	AvaloniaTest	D:\csharp\AvaloniaTest\App.axaml	1

**Comment by maxkatz6** [comment]
Hi!
Is it completely default app template with no changes?
